tools/nd_freeze

```diff
@@ -5,7 +5,7 @@
 # Install access to the NeuroDebian snapshot repository that allows access
 # to old packages based on dates and version numbers.
 #
-# Usage: nd_freeze [--keep-apt-sources|-k] [--no-downgrade|-n] date
+# Usage: nd_freeze [--keep-apt-sources --no-updates|-n] date
```

Comment:
Is dropping the `-k` here unintentional?  It looks like the short flag is still available.